crates/sp-consensus-spartan/Cargo.toml

```diff
@@ -14,11 +14,13 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 ring = { version = "0.16.20", optional = true }
-subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives", optional = true}
+
+# Subspace
+subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives", default-features = false }
```

Comment:
This doesn't need to be included in `no_std` environment and this crate will be gone very soon anyway, it is almost empty now

Reply:
Just want to gain more consistency since there is another same use case:

https://github.com/subspace/subspace/blob/9acbe21d968c3118209f6d6bb08dcfd695eb2bff/crates/subspace-archiving/Cargo.toml#L24

It's nothing different from the view of compiler, but we usually care more about `std` and `no_std` regarding the runtime related crates and in Substrate the default is typically `std`.